google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1123,6 +1125,9 @@ public Builder setHost(String host) {
     /** Enables gRPC-GCP extension with the default settings. */
     public Builder enableGrpcGcpExtension() {
       this.grpcGcpExtensionEnabled = true;
+      if (!userSpecifiedNumChannels) { // If numChannels is not set through options
+        this.numChannels = 8; // default when grpc-gcp channel pool is enabled
```

Comment:
Define a constant for 8, and use that to assign it here.

Reply:
Created constants `GRPC_GCP_ENABLED_DEFAULT_NUM_CHANNELS` for easier access.